src/box/index.h

```diff
@@ -459,23 +459,23 @@ key_validate(const struct index_def *index_def, enum iterator_type type,
 	     const char *key, uint32_t part_count);
 
 /**
- * Check that the supplied key is valid for a search in a unique
- * index (i.e. the key must be fully specified).
+ * Check that the index_def->key_def is valid for a search in
+ * a unique index (i.e. the key must be fully specified).
  * @retval 0  The key is valid.
  * @retval -1 The key is invalid.
  */
 int
-exact_key_validate(struct key_def *key_def, const char *key,
+exact_key_validate(struct index_def *index_def, const char *key,
 		   uint32_t part_count);
 
 /**
- * Check that the supplied key is valid for representing iterator
+ * Check that the index_def->cmp_def is valid for representing iterator
```

Comment:
That's wrong again.
The supplied key - is key argument.
I think you should write something like:
```
* Check that the supplied key is valid for representing iterator
* position, i.e. the key must be fully specified and correspond to
* index_def->cmp_def, with nulls are allowed.
* Note that since this function is designed checking a key for
* precise iterator position, it relies on index_def->cmp_def,
* in contrast to `key_validate` which relies on index_def->key_def.
* 
```

Reply:
Done.